harmony_regridding_service/regridding_service.py

```diff
@@ -589,7 +550,15 @@ def _clone_variables(
     """
     for dimension_name in dimensions:
         (s_var, t_var) = _copy_var_with_attrs(source_ds, target_ds, dimension_name)
-        t_var[:] = s_var[:]
+        try:
+            t_var[:] = s_var[:]
+        except IndexError as vlen_error:
+            # Handle snowflake metadata with vlen string.
+            if s_var.dtype == str and s_var.shape == ():
+                t_var[0] = s_var[0]
+            else:
+                logger.error('Unable to clone variable {s_var}')
+                raise SourceDataError('Unhandled variable clone') from vlen_error
```

Comment:
just what it says, strings can't be assigned with the slice operation[:] so handle that case ❄️

Reply:
This is the python-netcdf4 lib copying metadata on NetCDF.Datasets, this would be a snap with xarray probably, not sure about numpy.

It does just fail if it can't copy the variable.  IDK if that's appropriate or if we should write a warning? (that no one would see?)

Reply:
Discussed on a call - this makes sense as a future ticket. I'll mention this in a different comment in a moment.